tests/recovery/recovery_test/recovery_test.go

```diff
@@ -168,6 +172,172 @@ func TestSnapshotFixInvalidFiles(t *testing.T) {
 	require.NoError(t, err)
 }
 
+// TestConsistencyWhenKill9AfterModify will test the data consistency while it encountered kill -9 signal.
+func TestConsistencyWhenKill9AfterModify(t *testing.T) {
+	// assumption: the test is run on filesystem & not directly on object store
+	dataRepoPath := path.Join(*repoPathPrefix, dirPath, dataPath)
+
+	baseDir := t.TempDir()
+	if baseDir == "" {
+		t.FailNow()
+	}
+
+	bm, err := blobmanipulator.NewBlobManipulator(baseDir, dataRepoPath)
+	if err != nil {
+		if errors.Is(err, kopiarunner.ErrExeVariableNotSet) {
+			t.Skip("Skipping crash consistency tests because KOPIA_EXE is not set")
+		} else {
+			t.Skip("Error creating SnapshotTester:", err)
+		}
+	}
+
+	bm.DataRepoPath = dataRepoPath
+
+	// create a snapshot for initialized data
+	snapID, err := bm.SetUpSystemWithOneSnapshot()
+	if err != nil {
+		t.FailNow()
+	}
```

Comment:
How about using require package here too ? The advantage is that it'll actually also print the error which is good for debugging later.

```suggestion
	require.NoError(t, err)
```

Reply:
Fix it with `require.NoError(t, err)`